source/microbit/help.c

```diff
@@ -81,58 +81,65 @@ STATIC const mp_doc_t help_table_types[] = {
-    {&microbit_display_set_pixel_obj, "set_brightness(x, y, b) -- sets the brightness 'b' of the display pixel (x,y).\n"},
-    {&microbit_p0_obj, "Represents pin 0 on the gold edge connector.\n"},
-    {&microbit_p1_obj, "Represents pin 1 on the gold edge connector.\n"},
-    {&microbit_p2_obj, "Represents pin 2 on the gold edge connector.\n"},
-    {&microbit_p3_obj, "Represents pin 3 on the gold edge connector.\n"},
-    {&microbit_p4_obj, "Represents pin 4 on the gold edge connector.\n"},
-    {&microbit_p5_obj, "Represents pin 5 on the gold edge connector.\n"},
-    {&microbit_p6_obj, "Represents pin 6 on the gold edge connector.\n"},
-    {&microbit_p7_obj, "Represents pin 7 on the gold edge connector.\n"},
-    {&microbit_p8_obj, "Represents pin 8 on the gold edge connector.\n"},
-    {&microbit_p9_obj, "Represents pin 9 on the gold edge connector.\n"},
-    {&microbit_p10_obj, "Represents pin 10 on the gold edge connector.\n"},
-    {&microbit_p11_obj, "Represents pin 11 on the gold edge connector.\n"},
-    {&microbit_p12_obj, "Represents pin 12 on the gold edge connector.\n"},
-    {&microbit_p13_obj, "Represents pin 13 on the gold edge connector.\n"},
-    {&microbit_p14_obj, "Represents pin 14 on the gold edge connector.\n"},
-    {&microbit_p15_obj, "Represents pin 15 on the gold edge connector.\n"},
-    {&microbit_p16_obj, "Represents pin 16 on the gold edge connector.\n"},
-    {&microbit_p19_obj, "Represents pin 19 on the gold edge connector.\n"},
-    {&microbit_p20_obj, "Represents pin 20 on the gold edge connector.\n"},
-    {&microbit_pin_write_digital_obj, "write_digital(v) -- write a digital value v to the pin. Use 0 (lo) or 1 (hi).\n"},
-    {&microbit_pin_read_digital_obj, "read_digital() -- read the current digital value from the pin: 0 (lo) or 1 (hi).\n"},
-    {&microbit_pin_write_analog_obj, "write_analog(v) -- write an analog value v between 0..255 to the pin.\n"},
-    {&microbit_pin_read_analog_obj, "read_analog() -- read the current analog value from the pin: 0 - 65535.\n"},
-    {&microbit_pin_is_touched_obj, "is_touched() -- True if the pin is touched, otherwise False.\n"},
+    {&microbit_module, "Useful stuff to control the micro:bit hardware.\n"},
+    // System state objects
+    {&microbit_panic_obj, "Begin panic() mode (requires a reset).\n"},
+    {&microbit_random_obj, "Return a random(number) between 0 and 'number - 1'.\nrandom(10) returns a number between 0 and 9.\n"},
+    {&microbit_sleep_obj, "Put micro:bit to sleep(time) for some milliseconds (1 second = 1000 ms) of time.\nsleep(2000) gives micro:bit a 2 second nap.\n"},
+    {&microbit_running_time_obj, "Return running_time() in milliseconds since micro:bit's last reset.\n"},
+    // Accelerometer 3D orientation
+    {&microbit_accelerometer_obj, "Detect micro:bit's position (3D orientation).\n"},
+    {&microbit_accelerometer_get_x_obj, "Return micro:bit's X axis position.\n"},
```

Comment:
Can we address _how_ the X position is represented so kids grok what it means?

How about...

"Return a number showing micro:bit's X axis position." 

(Same of Y and Z)

TBH, not sure about this... comments..?